pkg/integration/tests/branch/sort_local_branches.go

```diff
@@ -6,29 +6,33 @@ import (
 )
 
 var SortLocalBranches = NewIntegrationTest(NewIntegrationTestArgs{
-	Description:  "Sort local branches alphabetically or by recency",
+	Description:  "Sort local branches by recency, date or alphabetically",
 	ExtraCmdArgs: []string{},
 	Skip:         false,
 	SetupConfig:  func(config *config.AppConfig) {},
 	SetupRepo: func(shell *Shell) {
-		shell.NewBranch("first")
-		shell.EmptyCommitWithDate("commit", "2023-04-07 10:00:00")
-		shell.NewBranch("second")
-		shell.EmptyCommitWithDate("commit", "2023-04-07 12:00:00")
-		shell.NewBranch("third")
-		shell.EmptyCommitWithDate("commit", "2023-04-07 11:00:00")
-		shell.Checkout("first")
+		shell.
+			NewBranch("master").
```

Comment:
No need to create the master branch, it's checked out by default. (Fixed in c5221ac915a5)